clients/node/examples/0-MakeRootIdentityAdmin.ts

```diff
@@ -0,0 +1,18 @@
+import { Manager, IdentityKeys } from 'iota-is-sdk';
+
+import { defaultManagerConfig } from './configuration';
+
+async function setup() {
+
+    // Create db connection
+    const manager = new Manager(defaultManagerConfig);
+
+    // Get root identity directly from db
+    let rootIdentityWithKeys: IdentityKeys = await manager.getRootIdentity();
```

Comment:
Why we don't create a new identity and then set the admin role for it? We really should not expose the root identity and even provide an example how to do it.

Reply:
For instance we could create an identity with a specific username, make it an admin and then always query this identity in later examples.

Reply:
I like the idea: however what do you propose?
Up to now we have two "Manager" methods that require database access:
- get root identity
- set role for identity
Should we leave only the second one? 
@dominic22

Reply:
Yes imo this should be enough 👍

Reply:
I tried.There are problems running example if you don't have the root identity. The reason is that you need an identity with a verifiable identity credential in order to create create other verifiable credentials. I don't know how to solve it without getting the root identity from database. I would leave the getRootIdentity in the Manager

Reply:
if you have admin rights you should be able to do so without a credential

Reply:
please log the jwt and make sure it has the correct rights

Reply:
createCredential needs an issued verifiable credential: even authorizing an admin (so obtaining right JWT) I'm not able to create credential.